### PR TITLE
podman create --hostname|-h same thing

### PR DESCRIPTION
**Pull request by baude**
Resolves: #624
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: cmd/podman/main.go [hunk 1]
```diff
@@ -26,7 +26,10 @@ func main() {
 	if reexec.Init() {
 		return
 	}
-
+	cli.HelpFlag = cli.BoolFlag{
```
Comment (TomSweeneyRedHat) [inline cmd/podman/main.go]: do you need this?  I think it's provided by default by urfavce/cli.

Comment (rhatdan) [reply cmd/podman/main.go]: I don't think you need this.

File: completions/bash/podman [hunk 2]
```diff
@@ -818,7 +815,6 @@ _podman_import() {
      "
     local boolean_options="
 	--help
-	-h
```
Comment (rhatdan) [inline completions/bash/podman]: I believe the -h will still work for --help for all commands except for podman run/create.  So these should not be removed?

Comment (TomSweeneyRedHat) [reply completions/bash/podman]: good point, the -h will still work for the commands other than run/create, I missed that here....

**Comment by baude** [comment]
closing

**Pull request closed without merging**